mwaa-local-env

```diff
@@ -14,6 +14,9 @@ display_help() {
    echo "build-image            Build Image Locally"
    echo "reset-db               Reset local PostgresDB container."
    echo "start                  Start Airflow local environment. (LocalExecutor, Using postgres DB)"
+   echo "start-silent           Start Airflow local environment as daemon. (LocalExecutor, Using postgres DB)"
+   echo "down                   Stop Airflow local environment. (LocalExecutor, Using postgres DB)"
+   echo "stop                   Shutdown Airflow local environment. (LocalExecutor, Using postgres DB)"
```

Comment:
start-daemon and stop-daemon are good additions and will be useful for non-debugging scenarios where you don't want to look at the CLI. I am not sure we want to de-abstract and use docker compose terms on the CLI. 

There should be a single stop-daemon command, the point is to decide if we should go the route of stop container or down container and remove them including volumes (-v). The second option is much cleaner and doesn't rack up a huge list of container definitions on the docker desktop dashboard - and leaves the least residue on configuration and networking.

@o-nikolas @dhegberg - please comment your thoughts on the above.

Reply:
@subashcanapathy Thank you for reviewing this. The main reasons for adding this is because I use the above syntax daily when developing in airflow. I do not want to continuously see printout of my logs by doing docker-compose -f compose-file up. I rather have it run in the background as daemon and see the log for each container on its own vs having it continuously print. it also helps with memory. Another command like down which I like to use since I do not have to constantly start from scratch. I can continue the docker where I left off much quickly. it's just a convenience I use on daily basis to do development. I believe this matches the use case for this repo.

Reply:
Yeah, it's tough with these types of things, more config means more interesting workflows but I also see what Subash is saying, that if used poorly the "pause" command could rack up residue.

With that being said though, the folks that are going to use this stuff likely know what they're doing when it comes to docker? So it'd probably be fine in most cases I reckon.